src/frontend/komponenter/Fagsak/Vedtak/VedtakContainer.tsx

```diff
@@ -72,27 +70,49 @@ const VedtakContainer: React.FC<IProps> = ({ behandling, fagsak }) => {
         behandling.behandlingsårsakstype ===
             Behandlingårsak.REVURDERING_FEILUTBETALT_BELØP_HELT_ELLER_DELVIS_BORTFALT;
 
-    const { hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(behandling.behandlingId);
+    const { erPerioderLike, hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(
+        behandling.behandlingId
+    );
+    const { erPerioderSlåttSammen, hentErPerioderSlåttSammen } = useErPerioderSlåttSammen(
+        behandling.behandlingId
+    );
+    const { slåSammenPerioder, feilmelding } = useSlåSammenPerioder(
+        behandling.behandlingId,
+        !erPerioderSlåttSammen
+    );
```

Comment:
Kunne disse to hookene vært slått sammen til en hook: `useSammenslåPerioder` ?

Slik det er nå, så avhenger jo `useSlåSammenPerioder` 100% av `useErPerioderSlåttSammen`. Jeg tror flyten ville vært enklere å følge dersom man samlet all koden for sammenslåing i en hook.

Reply:
Enig. Vi fikk løst dette ved å håndtere state i frontend, slik at vi slipper hooken `useErPerioderSlåttSammen`.